packages/api-docs-builder/utils/parseSlotsAndClasses.ts

```diff
@@ -59,11 +61,11 @@ export default function parseSlotsAndClasses({
     componentName,
     muiName,
   );
-  const slots = extractSlots(typescriptProject, componentName, classDefinitions);
+  const slots = extractSlots(typescriptProject, componentName, classDefinitions, slotInterfaceName);
 
-  const nonSlotClassDefinitions = classDefinitions
-    .filter((classDefinition) => !Object.keys(slots).includes(classDefinition.key))
-    .sort((a, b) => a.key.localeCompare(b.key));
+  const nonSlotClassDefinitions = classDefinitions.filter(
+    (classDefinition) => !Object.keys(slots).includes(classDefinition.key),
+  );
```

Comment:
I'm removing the sorting logic here such that sorting for classes and slots are both at the same place